Docs/source/dataanalysis/catalyst.rst

```diff
@@ -0,0 +1,167 @@
+
+
+Enabling Catalyst
+-----------------
+In order to use Catalyst with WarpX, you must `build Catalyst 2 <https://catalyst-in-situ.readthedocs.io/en/latest/build_and_install.html>`_ and `build <https://github.com/Kitware/ParaView/blob/master/Documentation/dev/build.md>`__ or `install <https://www.paraview.org/download/>`__ ParaView 5.9+. Afterward, AMReX must be built with ``AMReX_CONDUIT=TRUE``,
+``AMReX_CATALYST=TRUE``, ``Conduit_DIR=/path/to/conduit``, and ``Catalyst_DIR=/path/to/catalyst`` (``/path/to/catalyst`` should be the directory containing ``catalyst-config.cmake``, not the path to the implementation).
+
+Once AMReX is appropriately built, WarpX can be built with the following options:
+
+.. code-block:: cmake
+
+    WarpX_amrex_internal=FALSE
+    AMReX_DIR="/path/to/amrex/build"
+
+If they cannot be found, ``Conduit_DIR`` and ``Catalyst_DIR`` will have to be set again. Ensure that AMReX is built with all required options, some common ones being:
+
+.. code-block:: cmake
+
+    AMReX_MPI=TRUE
+    AMReX_MPI_THREAD_MULTIPLE=TRUE
+    AMReX_LINEAR_SOLVERS=TRUE
+    AMReX_PARTICLES=TRUE
+    AMReX_PARTICLES_PRECISION=DOUBLE
+    AMReX_PIC=TRUE
+    AMReX_TINY_PROFILE=TRUE
+
+
+Inputs File Configuration
+-------------------------
+Once WarpX has been compiled with Catalyst support, it will need to be enabled and configured at runtime.
+This is done using our usual inputs file (read with ``amrex::ParmParse``).
+The supported parameters are part of the :ref:`FullDiagnostics <running-cpp-parameters-diagnostics>` with ``<diag_name>.format`` parameter set to ``catalyst``.
+
+In addition to configuring the diagnostics, the following parameters must be included:
+    * ``catalyst.script_paths``: The locations of the pipeline scripts, separated by either a colon or semicolon (e.g. ``/path/to/script1.py;/path/to/script2.py``).
+    * ``catalyst.implementation`` (default ``paraview``): The name of the implementation being used (case sensitive).
+    * ``catalyst.implementation_search_paths``: The locations to search for the given implementation. The specific file being searched for will be ``catalyst_{implementation}.so``.
+
+Because the scripts and implementations are global, Catalyst does not benefit from nor differentiate between multiple diagnostics.
+
+
+Visualization/Analysis Pipeline Configuration
+---------------------------------------------
+Catalyst uses the files specified in ``catalyst.script_paths`` to run all analysis.
+
+The following script, :code:`simple_catalyst_pipeline.py`, automatically detects the type of data for both the mesh and particles, then creates an extractor for them. In most
+cases, these will be saved as ``.VTPC`` files which can be read with the ``XML Partitioned Dataset Collection Reader``.
+
+.. code-block:: python
+
+    from paraview.simple import *
+    from paraview import catalyst
+
+    # Helper function
+    def create_extractor(data_node, filename="Dataset"):
```

Comment:
For `create_extractor` I cannot follow from the doc string what it does yet - can add a more verbose string here?
Feel free to use the usual
```
```suggestion
    def create_extractor(data_node, filename="Dataset"):
        """Here goes the doc string block"""
```
syntax.